proto/stat_var.proto

```diff
@@ -232,4 +232,22 @@ message GetStatVarSummaryRequest {
 }
 message GetStatVarSummaryResponse {
   map<string, StatVarSummary> stat_var_summary = 1;
+}
+
+message GetStatVarMatchRequest {
+  // The property value pair. Ex
+  // {gender: Male}, {statType: measuredValue}
+  // To improve this, can explicitly specify "statType", "measuredProperty",
+  // and constraint properties. Then the match could have must-match terms and
+  // optional match terms that have weights. That would greatly reduce the
+  // number of returned match stat vars.
+  map<string, string> property_value = 1;
```

Comment:
What is multiple values?

Reply:
@nopper Can you confirm if multiple values could exist for one property?

This is a base version of the API which we hope to iterate. As in the comment above, more information could be added in the request, so I am trying to make it simple for now and it would need more changes anyway.

Reply:
The current prototype only emits a single value per property. So it should be fine like this at the moment.